pkg/graphgen/edgeless.go

```diff
@@ -0,0 +1,39 @@
+package graphgen
+
+import (
+	"strconv"
+
+	"github.com/arangodb/feed/pkg/datagen"
+)
+
+type EdgelessGraph struct {
+	Size   uint64 // number of vertices
+	Prefix string
+}
+
+func (g EdgelessGraph) MakeGraphGenerator() (GraphGenerator, error) {
```

Comment:
Here is the target the object, previously it was a pointer to the object in other cases. Is this intentional?

Reply:
No, I'll change it.